### PR TITLE
Use the correct ShadowPalette for d2k paradrops

### PR DESCRIPTION
**Pull request by abcdefg30**
Followup from #8719.
Fixes this crash when paradroping units on bleed:

```
Dune 2000 Mod at Version {DEV_VERSION}
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.34209
Exception of type `System.InvalidOperationException`: Palette `player` does not exist
   at OpenRA.Graphics.HardwarePalette.GetPalette(String name) in \OpenRA\OpenRA.Game\Graphics\HardwarePalette.cs:Line 46.
   at OpenRA.Graphics.WorldRenderer.CreatePaletteReference(String name) in \OpenRA\OpenRA.Game\Graphics\WorldRenderer.cs:Line 65.
   at OpenRA.Exts.GetOrAdd[K,V](Dictionary`2 d, K k, Func`2 createFn) in \OpenRA\OpenRA.Game\Exts.cs:Line 129.
   at OpenRA.Graphics.WorldRenderer.Palette(String name) in \OpenRA\OpenRA.Game\Graphics\WorldRenderer.cs:Line 69.
   at OpenRA.Mods.Common.Traits.WithParachute.<Render>d__e.MoveNext()
   at OpenRA.Actor.<Renderables>d__d.MoveNext() in \OpenRA\OpenRA.Game\Actor.cs:Line 156.
   at System.Linq.Enumerable.<SelectManyIterator>d__14`2.MoveNext()
   at System.Linq.Enumerable.<ConcatIterator>d__71`1.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.OrderedEnumerable`1.<GetEnumerator>d__0.MoveNext()
   at System.Linq.Enumerable.<ConcatIterator>d__71`1.MoveNext()
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at OpenRA.Graphics.WorldRenderer.GenerateRenderables() in \OpenRA\OpenRA.Game\Graphics\WorldRenderer.cs:Line 117.
   at OpenRA.Graphics.WorldRenderer.Draw() in \OpenRA\OpenRA.Game\Graphics\WorldRenderer.cs:Line 134.
   at OpenRA.Game.RenderTick() in \OpenRA\OpenRA.Game\Game.cs:Line 525.
   at OpenRA.Game.Loop() in \OpenRA\OpenRA.Game\Game.cs:Line 653.
   at OpenRA.Game.Run() in \OpenRA\OpenRA.Game\Game.cs:Line 671.
   at OpenRA.Program.Run(String[] args) in \OpenRA\OpenRA.Game\Support\Program.cs:Line 111.
   at OpenRA.Program.Main(String[] args) in \OpenRA\OpenRA.Game\Support\Program.cs:Line 39.
```

### CODE REVIEW

**Comment by pchote** [comment]
Can we please fix the default instead?  Defaulting `ShadowPalette` to `player` doesn't make a lot of sense.

**Comment by abcdefg30** [comment]
Updated.

File: mods/ra/rules/defaults.yaml [hunk 1]
```diff
@@ -128,7 +128,12 @@
 	WithParachute:
 		UpgradeTypes: parachute
 		UpgradeMinEnabledLevel: 1
+		ShadowPalette: player
```
Comment (pchote) [inline mods/ra/rules/defaults.yaml]: Why is this necessary?

**Comment by obrakmann** [comment]
The scripted infantry paradrops in at least the beginning of Survival01 and Intervention don't seem to work anymore, either. (not saying that it's this PR's fault, but that it needs to be fixed.)

**Comment by obrakmann** [comment]
I figured it out. The scripted parachuting doesn't work because in #8339, the parachute was changed from an effect to an activity. However, idle triggers in the scripts cause the units to already have an activity, so the `Parachute` activity is queued and would only run once the initial activity (usually a `Hunt` or `AttackMove`) is finished.

File: mods/ra/rules/defaults.yaml (hunk 1 above)
Comment (abcdefg30) [inline mods/ra/rules/defaults.yaml]: Doesn't seem like we need this.

**Comment by abcdefg30** [comment]
Updated and removed `ShadowPalette: player`.

**Comment by obrakmann** [comment]
Fixed the scripted paradrops in https://github.com/abcdefg30/OpenRA/pull/6.

**Comment by abcdefg30** [comment]
Merged.

**Comment by obrakmann** [comment]
Both d2k and ra paradrops (both infantry and vehicles) work again, :+1:

**Comment by Mailaender** [comment]
Paratroopers work in skirmish and Survival 02 as well as Dune 2000. :white_check_mark:
